### PR TITLE
Add Clear Button

### LINKED ISSUE #11: some way to clear drawing area

### PR DESCRIPTION
**Pull request by PurpleMyst**
picture is just a copyright-free placeholder :p

this commit also fixes a bug where the "change" event was not emitted when undoing

closes #11

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,6 +82,8 @@
                 <input type="image" class="pic-button color-button" src="pics/1x1-transparent.png" style="background-color: blue;" />
                 <input type="image" class="pic-button color-button" src="pics/1x1-transparent.png" style="background-color: green;" />
                 <input type="image" class="pic-button color-button" src="pics/1x1-transparent.png" style="background-color: magenta;" />
+                <hr class="button-separator" />
+                <input type="image" title="Undo All" class="pic-button" id="undo-all-button" src="pics/undoall.png " />
```
Comment (Akuli) [inline index.html:86]:
```suggestion
                <input type="image" title="Clear" class="pic-button" id="undo-all-button" src="pics/undoall.png " />
```

**Review by Akuli — COMMENTED** [review]
nice, one little thing

**Comment by Akuli** [comment]
maybe rename `undoAll` --> `clear` in code as well to be consistent with the button tooltip that now says clear?

**Review by Akuli — COMMENTED** [review]
Code looks good but the icon isn't great. I don't think I have seen anyone wiping a blackboard or whiteboard with a broom, which is why I drew a mop icon (akuli-canvas-clear branch), but that is also bad because it looks like a stick pointing at a whiteboard.

**Comment by PurpleMyst** [comment]
Maybe a trash can or maybe a rubbing alcohol bottle? The former works for the "paper" metaphor and the latter works for the "whiteboard" metaphor. But still, I think the trash can is the most universal: we could even open the lid on mouse over :p

**Pull request merged**